ghc-mod.cabal

```diff
@@ -201,6 +205,8 @@ Executable ghc-mod
                       , mtl       < 2.3 && >= 2.0
                       , ghc       < 7.11
                       , fclabels  ==2.0.*
+                      , optparse-applicative ==0.11.*
+                      , ansi-wl-pprint ==0.6.*
```

Comment:
Ah, ok. Let's just get rid of the other one then. I'll merge this anyways though, we can do that later.